packages/ui/src/component/advanced/Button.ts

```diff
@@ -0,0 +1,41 @@
+import { PointerEventData, SafeLoopArray } from "@galacean/engine";
+import { UIInteractive } from "../interactive/UIInteractive";
+
+export class Button extends UIInteractive {
```

Comment:
1、Is it necessary to support multiple types of Transition simultaneously
2、Is it more appropriate for various transition-related properties to be exposed directly from the Button or to be accessed through Transition